CHANGELOG.md

```diff
@@ -1,5 +1,42 @@
 # Changelog
 
+## [0.63.48](https://github.com/brave/brave-browser/releases/tag/v0.63.48)
+
+ - Enable Ads on Release Channel.([#4209](https://github.com/brave/brave-browser/issues/4209))
+ - Loss of "Clickable area" on new tab page tiles.([#3487](https://github.com/brave/brave-browser/issues/3487))
+ - Option to clear browsing data on exit.([#492](https://github.com/brave/brave-browser/issues/492))
+ - Use global top navigation bar for settings, rewards, history, bookmarks, downloads.([#956](https://github.com/brave/brave-browser/issues/956))
+ - Add chrome://rewards-internals to assist with support.([#1174](https://github.com/brave/brave-browser/issues/1174))
+ - Beta/Dev build UI issues on Manjaro/Arch Linux .([#1555](https://github.com/brave/brave-browser/issues/1555))
+ - Widevine prompt needs to be more obvious to the user.([#1940](https://github.com/brave/brave-browser/issues/1940))
+ - Special characters in YT channel names not appearing correctly under rewards panel.([#2028](https://github.com/brave/brave-browser/issues/2028))
+ - Blocking functionality on www.wsj.com.([#2286](https://github.com/brave/brave-browser/issues/2286))
+ - Mobile.twitter.com right click hang.([#2410](https://github.com/brave/brave-browser/issues/2410))
+ - Sync sends bookmarks even when chain is not fully created.([#2687](https://github.com/brave/brave-browser/issues/2687))
+ - A-c table not being updated correctly - follow up to 2882.([#2911](https://github.com/brave/brave-browser/issues/2911))
+ - Videos don't play on videoportal.joj.sk.([#2931](https://github.com/brave/brave-browser/issues/2931))
+ - Unable to exclude sites from the a-c table.([#2943](https://github.com/brave/brave-browser/issues/2943))
+ - Crash when trying to open brave://sync in tor window.([#3003](https://github.com/brave/brave-browser/issues/3003))
+ - Restored sites got deleted after page reload, when rewards page opened in multiple windows.([#3264](https://github.com/brave/brave-browser/issues/3264))
+ - Tipping panel view not showing on some YouTube channels.([#3335](https://github.com/brave/brave-browser/issues/3335))
+ - Bookmark and pin icon behavior is not correct for top sites (on new tab page).([#3347](https://github.com/brave/brave-browser/issues/3347))
+ - Checkboxes should use Brave color palette.([#3377](https://github.com/brave/brave-browser/issues/3377))
+ - Buttons should use Brave brand style.([#3378](https://github.com/brave/brave-browser/issues/3378))
+ - Cleanup showing sync & reward menu in app menu model.([#3410](https://github.com/brave/brave-browser/issues/3410))
```

Comment:
Display sync and Brave Rewards as menu items for normal and private windows and hide them for Tor windows.